homeassistant/components/light/zwave.py

```diff
@@ -61,7 +61,7 @@ def get_device(node, values, node_config, **kwargs):
 def brightness_state(value):
     """Return the brightness and state."""
     if value.data > 0:
-        return round((value.data / 99) * 255, 0), STATE_ON
+        return int((value.data / 99) * 255), STATE_ON
```

Comment:
use `round((value.data / 99) * 255)` instead.
It will return an it but will round instead of truncate

Reply:
I've changed that, although I don't think that it would have been that big of a difference (only 0.5/255 max deviation), round might not return to [expected value](https://docs.python.org/3/library/functions.html#round) and `int` is used later for the backwards conversion.